Update browser-features.md (#1303)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/browser-features.md

*   Most features work the same in WebView2 and Microsoft Edge.  If a feature doesn't make sense in the context of WebView2 or for other reasons, the feature is modified or turned off. 
*   WebView2 features don't include Microsoft Edge branding.  
    

> ## Browser Features  

Reviewer: ```suggestion
## Browser features  
```

Author: done

| Read Aloud | Off | No | This feature is turned off.  |  
| Smart Screen | On`*` | No | `*` The UI for this feature has been removed, however the underlying functionality is still available.  Additionally, you may turn off Smart Screen using a command-line switch.  |  
| Translate | Off | No | This feature is turned off.  |  
| Tracking Prevention | On`*` | No | `*` The UI for this feature has been removed, however the underlying functionality is still available.  Tracking prevention is always set to balanced.|  
| Profile and Identity | Off | No | The feature that syncs your favorites, cookies, and so on, is turned off.  | 
| Windows Defender Application Guard | Off | No | This feature is turned off.  |  
| edge:// URLs | Review Details | No | Settings for the Microsoft Edge browser are on `edge://` URLs.  Because most of these webpages have Microsoft Edge branding or don't make sense within the context of WebView2, some of these URLs are turned off.  For more information, navigate to [Blocked internal URLs](#blocked-internal-urls).  |  

> ## Platform Features

Reviewer: ```suggestion
## Web platform features
```

Author: done

> The following table displays the WebView2 platform features that are currently removed.

Reviewer: ```suggestion
The following table displays the WebView2 web platform features that are currently unavailable.
```

Author: done

| Feature | Default state | Configurable | Details |  
|:--- |:--- |:--- | :--- |  
| Push Notifications | Off | No | This feature is not implemented in WebView2.  For more information, navigate to [Add support for HTML5 Notification API (#308)][GithubMicrosoftedgeWebview2feedbackIssues308]. |  

> | Web Payment API | Off | No | This feature is turned off. | 

Reviewer: I suggest removing columns 2 and 3.

Author: done
